### PR TITLE
feature/code-coverage-tool

### PR DESCRIPTION
**Pull request by andytaylorkainos**
This PR is a wrapper for running the code-coverage tool and implements CircleCI failures if overall coverage is < 80%

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -70,7 +75,7 @@ jobs:
...
           command: pipenv run ./manage.py test --parallel
 
-      # Linting: dodgy, mccabe, pep8, profile-validator, pyflakes, pylint
+      # Code Coverage
       - run:
-          name: Run Linting
-          command: pipenv run prospector
\ No newline at end of file
+          name: Run Code Coverage
+          command: pipenv run ./manage.py coverage --threshold=80
```
Comment (markm5A8Z9F43913A) [inline .circleci/config.yml]: If we're specifying it here, why are we specifying it in self.DEFAULT_THRESHOLD too?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,23 @@ Service for handling backend calls in LITE.
...
 [Running locally without Docker](docs/without-docker.md)
 
+## Running Tests
+
+- `pipenv run ./manage.py test` will run all tests
+- `pipenv run ./manage.py test cases` will run the `cases` module tests
+
+## Running Code Coverage
+
+- `pipenv run ./manage.py coverage <positional_arg_1> <positional_arg_2>`
```
[the remark this answers is not in the document]
Comment (andytaylorkainos) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -32,6 +32,23 @@ Service for handling backend calls in LITE.
...
+
+1. The first positional argument implies what module you want to collect coverage for:
+    - `pipenv run ./manage.py coverage cases` will collect coverage on the `cases` module from running the `cases` tests
+2. The second positional argument implies what tests to run in order to collect coverage for the given module:
+    - `pipenv run ./manage.py coverage cases all` will collect coverage on the `cases` module from running all tests
+    - `pipenv run ./manage.py coverage cases queues` will collect coverage on the `cases` module from running the `queues` tests
+3. Providing no positional arguments implies that you want to collect coverage for all modules from running all tests:
```
[the remark this answers is not in the document]
Comment (andytaylorkainos) [reply README.md]: Done

File: static/management/commands/coverage.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
+from subprocess import call as execute_bash_command
+from sys import exit as sys_exit
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
```
[the remark this answers is not in the document]
Comment (andytaylorkainos) [reply static/management/commands/coverage.py]: Done

**Pull request merged**